domains/certificates/Query.API/API.AppTests/ContractTests.cs

```diff
@@ -61,11 +71,11 @@ public async Task CreateContract_GsrnAlreadyExistsInDb_Conflict()
 
         var body = new { gsrn, startDate = DateTimeOffset.Now.ToUnixTimeSeconds() };
 
-        var response = await client.PostAsJsonAsync("api/certificates/contracts", body);
+        using var response = await client.PostAsJsonAsync("api/certificates/contracts", body);
         response.StatusCode.Should().Be(HttpStatusCode.Created);
 
-        response = await client.PostAsJsonAsync("api/certificates/contracts", body);
-        response.StatusCode.Should().Be(HttpStatusCode.Conflict);
+        using var response2 = await client.PostAsJsonAsync("api/certificates/contracts", body);
```

Comment:
Could this be renamed to something like `conflictResponse` ? just from reading this text it is hard to figure out what response2 is compared to response

Reply:
Fixed in 640a5f61d68372a04019367232d71da60ab16e4d